relay-general/src/pii/convert.rs

```diff
@@ -445,12 +438,15 @@ THd+9FBxiHLGXNKhG/FRSyREXEt+NyYIf/0cyByc9tNksat794ddUqnLOg0vwSkv
         assert_annotated_snapshot!(data);
     }
 
+    /// Checks that any fields containing an IP-address is scrubbed.
+    /// Even if it doesn't make sense for there to be an IP-address
+    /// such as in the username field.
     #[test]
     fn test_user_ip_stripped() {
         let mut data = Event::from_value(
             serde_json::json!({
                 "user": {
-                    "username": "secret",
+                    "username": "73.133.27.120",
```

Comment:
it would be great to test if the scrubbing works also in , e.g. `breadcrumbs` - if you add the message with ip inside , as it's done in test https://github.com/getsentry/relay/blob/3b42784ba9359db65bd890b8d628414e3201de8c/relay-general/src/pii/convert.rs#L1197 
the protocol for `breadcrumbs` is in https://github.com/getsentry/relay/blob/3b42784ba9359db65bd890b8d628414e3201de8c/relay-general/src/protocol/breadcrumb.rs#L49 where you can see which fields are PII